openshift/ci-operator/generate-dockerfiles.sh

```diff
@@ -19,3 +19,8 @@ function generate_dockefiles() {
 }
 
 generate_dockefiles $@
+
+# shellcheck disable=SC2002
+go_version=$(grep "^go.*" "go.mod" | awk '{print $2}')
```

Comment:
+1

(with the assumption we have that one on our registry, which is high-likely)